wit/consumer.wit

```diff
@@ -1,27 +1,17 @@
+/// The consumer interface allows a guest to dynamically update its subscriptions and configuration
 interface consumer {
```

Comment:
Are the guest and consumer interfaces separate due to only wanting to export the handler function and not both handler and update-guest-configuration?

Reply:
Now that you say it, they should probably just be combined

Reply:
Oh wait, that's right. They are separate because a host exports the configuration function to the guest, whereas the guest exports the handler to the host

Reply:
Thank you for the clarification. Just wanted to make sure I understood the intent.